core/src/TopNavDropDown.html

```diff
@@ -123,6 +128,22 @@ <h5 class="fd-action-bar__title fd-has-type-1">{getNodeLabel(node)}</h5>
       },
       getNodeLabel(node) {
         return LuigiI18N.getTranslation(node.label);
+      },
+      prepareForTests(...parts) {
```

Comment:
Please move this code (from here and all the other places) to a generic helper and call the helper function instead.